CV Series/5_Package and deploy CV/cv.css

```diff
@@ -0,0 +1,145 @@
+body {
```

Comment:
Can you please move all the "demo" files into a 'CV' directory? like the others. The you can leave the presentation in this main directory.